workflows/rnaseq/downstream/functional-enrichment.Rmd

```diff
@@ -48,10 +88,14 @@ pvalueCutoff <- 1
 qvalueCutoff <- 1
 go.uni <- NULL
 path.uni <- NULL
-kegg.organism <- 'dme'
+kegg.organism <- 'mmu'
 RUN.REACTOME <- FALSE
-reactome.organism <- 'fly'
+reactome.organism <- 'mouse'
+min.gene.set.size <- 50
```

Comment:
I think this is a reasonable number and idea, but should be included in the PR (and eventually added to the changelog as well).